doc/source/whatsnew/v0.24.0.txt

```diff
@@ -182,6 +182,7 @@ Other Enhancements
 - :func:`to_timedelta` now supports iso-formated timedelta strings (:issue:`21877`)
 - :class:`Series` and :class:`DataFrame` now support :class:`Iterable` in constructor (:issue:`2193`)
 - :class:`DatetimeIndex` gained :attr:`DatetimeIndex.timetz` attribute. Returns local time with timezone information. (:issue:`21358`)
+- Added :meth:`Resampler.quantile` (:issue:`15023`).
```

Comment:
can you say that  ``Series.resample`` and ``DataFrame.resample`` have gained the ``Resample.quantile()`` method (use references for .resample)